docs/workflow/snippets/sdk-net-limitations-warning.mdx

```diff
@@ -0,0 +1,3 @@
+:::note
+We recommend that you only implement the <code>IActivityHandler</code> interface examples provided in the Developer Center. These examples have been verified and are provided to help you customize your Geocortex product to suit your needs. If you choose to use classes or functions not documented here, you may break functionality and/or experience unexpected results.
```

Comment:
"If you choose to use classes or functions not documented here" sounds super broad. What we mean is that you should not reference other classes and functions from the various Geocortex Workflow assemblies .

Reply:
Changed

Reply:
Am I crazy or does this now read like "you should only implement these exact examples that we are providing on the DevCenter. If you do anything else you are taking a risk." 

We want to make it clear that IActivityHandler is fully supported, but all the other public classes and functions **from Geocortex Workflow** are not.

Reply:
That's what I had meant by "from the various Geocortex Workflow assemblies". I'm open to suggestions if there's a better way to word it.

Reply:
Turns out I was crazy, and looking at the outdated code. All good.